gradle/libs.versions.toml

```diff
@@ -24,6 +24,8 @@ dependencyAnalysis = { id = "com.autonomousapps.dependency-analysis", version.re
 detekt = { id = "io.gitlab.arturbosch.detekt", version.ref = "detekt" }
 dokka = { id = "org.jetbrains.dokka", version.ref = "dokka" }
 ksp = { id = "com.google.devtools.ksp", version.ref = "ksp" }
+# Here to trigger Renovate updates
+kotlinksp = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
```

Comment:
Questions:

1. I'm curious why we're naming the jvm artifact as ksp.
2. kolinksp is all lowercase, and not kotlin-ksp. Is this intentional?

Reply:
derp, this should be kotlin.jvm

Reply:
```suggestion
kotlin.jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
```

Reply:
do we want dot in the name? Perhaps kotlinJvm or kotlin-jvm?